recipe/meta.yaml

```diff
@@ -63,6 +64,10 @@ about:
   dev_url: https://github.com/pytest-dev/pytest/
 
 extra:
+  skip-lints:
+    # Without `packaging`, pytest errors out with:
+    #   pytest <version> requires packaging, which is not installed.
+    - python_build_tool_in_run
```

Comment:
`packaging` should be excluded from that lint rule. `packaging` isn't a build tool. It's a widely used library that just happens to be used by build tools.

Do you mind creating an issue in the anaconda-linter for that please?

Reply:
imports packaging would not fix the linter problem ?

Reply:
@JeanChristopheMorinPerso As an expert in the subject, would you mind raising that issue with the linter and maybe provide examples. I don't want to play "telephone" and miss any important details/examples. Anyone can submit issues to the [linter project](https://github.com/anaconda-distribution/anaconda-linter/issues)

Reply:
Yep. In fact, I already created a PR: https://github.com/anaconda-distribution/anaconda-linter/pull/229